package.json

```diff
@@ -9,6 +9,7 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "@expo/vector-icons": "^14.0.4",
```

Comment:
don't you know how react applications build? (I need information here)
does it includ all dependencies to a build, or automatically link deps by imports, or manually link by expo/bable builder?

Reply:
Upd: seems It doesn't important whatever section of deps will be for react-native project, as there babel exists